server.js

```diff
@@ -20,8 +21,12 @@ function fullUrl (req) {
 const listener = app.listen(port, () => {
     console.log(`Server listening at port ${listener.address().port}`);
 });
+const origin = 'https://privacy-test-pages.glitch.me';
 
 app.use(express.json());
+app.use(json({
+    type: 'application/csp-report'
+}));
```

Comment:
I'm guessing that this automatically parses body of requests matching this mime type? If so, can we add a comment describing what this does?